admin-ui/plugins/fido/helper/utils.ts

```diff
@@ -99,16 +118,16 @@ const createFidoConfigPayload = ({ fidoConfiguration, data, type }: any) => {
     payload.metricReporterEnabled = data.metricReporterEnabled
     payload.metricReporterInterval = data.metricReporterInterval
     payload.metricReporterKeepDataDays = data.metricReporterKeepDataDays
-    payload.personCustomObjectClassList = data.personCustomObjectClassList.map((item: any) =>
-      item?.value ? item?.value : item,
-    )
+    payload.personCustomObjectClassList = (
+      data.personCustomObjectClassList as Array<{ value?: string } | string>
+    ).map((item) => (typeof item === 'object' && item?.value ? item.value : (item as string)))
 
     if (payload.fido2Configuration) {
       payload.fido2Configuration.hints = data.hints || []
     }
   }
 
-  const opts: Record<string, any> = {}
+  const opts: Record<string, unknown> = {}
   opts['appConfiguration1'] = payload
 
   return opts
```

Comment:
Incorrect request key: use appConfiguration

The API expects `{ appConfiguration }`, not `appConfiguration1`. Current shape will 400/422 server-side.

Apply:

```diff
-  const opts: Record<string, unknown> = {}
-  opts['appConfiguration1'] = payload
-
-  return opts
+  const opts: Record<string, unknown> = {}
+  opts['appConfiguration'] = payload
+  return opts
```